https://bugs.webkit.org/show_bug.cgi?id=39847

additionally, also on Darwin8 glib stuff includes system headers that
use isascii, so we can't have it die on that.

--- JavaScriptCore/wtf/FastMalloc.cpp
+++ JavaScriptCore/wtf/FastMalloc.cpp
@@ -1381,14 +1381,12 @@
   // Bytes allocated from system
   uint64_t system_bytes_;
 
-#if USE_BACKGROUND_THREAD_TO_SCAVENGE_MEMORY
   // Number of pages kept in free lists that are still committed.
   Length free_committed_pages_;
 
   // Minimum number of free committed pages since last scavenge. (Can be 0 if
   // we've committed new pages since the last scavenge.)
   Length min_free_committed_pages_since_last_scavenge_;
-#endif
 
   bool GrowHeap(Length n);
 
--- WebCore/config.h
+++ WebCore/config.h
@@ -125,7 +125,7 @@
 // this breaks compilation of <QFontDatabase>, at least, so turn it off for now
 // Also generates errors on wx on Windows, presumably because these functions
 // are used from wx headers.
-#if !PLATFORM(QT) && !PLATFORM(WX) && !PLATFORM(CHROMIUM)
+#if !PLATFORM(QT) && !PLATFORM(WX) && !PLATFORM(CHROMIUM) && !defined(BUILDING_ON_TIGER)
 #include <wtf/DisallowCType.h>
 #endif
 
